src/component.js

```diff
@@ -38,7 +38,7 @@ function createSubmitHandler (hostedFieldsInstance, orderIdFunction) : Function
         logger.track({
           comp:                                'hostedpayment',
           // risk_correlation_id: 'TODO',
-          card_brand:                          payload && payload.payment_source.card.card_type,
+          card_brand:                          payload && payload.payment_source && payload.payment_source.card.card_type,
```

Comment:
Still think we need to have access to the card details from the original validate call. Anyway for us to get it?

Reply:
Wouldn't want the card details at this layer right? Then they're exposed to the merchant

Reply:
It's not the raw card details I'm talking about, just the card type information, which is not in PCI scope.

If we don't go through the 3ds flow, we get that information via the validate-payment endpoint. If we do, that info is lost and can't be sent in the analytic. (why we even need to send this info on the client though, is beyond me, seems like we should already have it on the server)